testing/hiawatha/hiawatha.initd

```diff
@@ -2,12 +2,21 @@
 
 supervisor=supervise-daemon
 
-name="Webserver"
-description="Secure and Advanced Hiawatha Webserver"
+name="Hiawatha webserver"
+description="Secure and advanced webserver"
 
 command=/usr/sbin/hiawatha
-command_args_foreground="-d"
+command_args_foreground="-d 1>/dev/null"
+pidfile=/var/run/hiawatha.pid
```

Comment:
pidfile isn't required by supervise-daemon, the whole reason to use supervise daemon is to not have to use the disgusting hack that is pidfiles.

Reply:
It seems to be required:

```
root@proxy [~]# service hiawatha start
 * Caching service dependencies ... [ ok ]
 * Starting Hiawatha webserver ... * supervise-daemon: --pidfile must be specified
 * failed to start Hiawatha webserver
                                                                                                                                              [ !! ]
 * ERROR: hiawatha failed to start
```
with `pidfile` set it starts